projects/rero/ng-core/src/lib/record/detail/detail.component.html

```diff
@@ -19,16 +19,28 @@
     <div class="col-9">
         <ng-template ngCoreRecordDetail></ng-template>
     </div>
-    <div class="col-3" *ngIf="record">
-        <a href="javascript:void(0);" class="btn btn-sm btn-secondary" routerLink="../../edit/{{ record.metadata.pid }}">
+    <div class="col-3 text-right" *ngIf="record">
+        <a href class="btn btn-sm btn-primary"
+            routerLink="../../edit/{{ record.metadata.pid }}" *ngIf="canUpdateResult">
```

Comment:
`canUpdateResult.can`? or better `canUpdateResult && canUpdateResult.can`

Reply:
canUpdateResult is not an object, but directly resolved as a boolean. It's different in regard to canDeleteResult because the latter returns an object with the boolean and eventually a message.

Reply:
ok, I saw your comment later on, which is relevent. Il will create an interface ActionStatus which returns always an object for all actions